packages/astro/src/@types/astro.ts

```diff
@@ -1010,6 +1012,71 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		service: ImageServiceConfig;
+
+		/**
+		 * @docs
+		 * @name image.domains (Experimental)
+		 * @type {Array<string>}
+		 * @default `{domains: []}`
+		 * @version 2.9.2
+		 * @description
+		 * Defines a list of domains allowed to be optimized locally
+		 *
+		 * The value should be an array of domain strings of images allowed to be optimized locally.
+		 * Wildcards are not permitted (see image.remotePatterns).
+		 *
+		 * ```js
+		 * {
+		 *   image: {
+		 *     // Example: Enable the Sharp-based image service
+		 *     domains: ['astro.build'],
+		 *   },
+		 * }
```

Comment:
```suggestion
		 * // astro.config.mjs
		 * {
		 *   image: {
		 *     // Example: Allow remote image optimization from a single domain
		 *     domains: ['astro.build'],
		 *   },
		 * }
```
I'll also point out that the changeset shows a trailing slash, and this example doesn't, so just make them both consistent with whatever it should be!